stdlib/src/builtin/math.mojo

```diff
@@ -235,6 +235,9 @@ trait Roundable:
     fn __round__(self) -> Self:
         ...
 
+    fn __round__(self, ndigits: IntLiteral) -> Self:
```

Comment:
Requiring `ndigits` to be `IntLiteral` will make this impossible to use at runtime. Can you please change this to accept just an `Int`?

Reply:
Done: https://github.com/msaelices/mojo/commit/9de5acca6db1dad73f160a084687cdfaba9f5ed4